### PR TITLE
GIVCAMP-131 | Replace dcnb with cnb for speed

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Replace dcnb with cnb because we don't need the dedupe the way we're using it
- Author says dedupe version is much slower. Looking at the benchmark, we don't get much advantage using it over classname package if we use the dcnb version
https://github.com/xobotyi/cnbuilder/tree/master/benchmark

# Review By (Date)
- Retro

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at the preview and see that things still look the same

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-131

### CODE REVIEW

File: src/components/Bracket/Bracket.styles.ts [hunk 1]
```diff
@@ -11,8 +11,8 @@ export const colors = {
 };
 export type ColorType = keyof typeof colors;
 
-export const curve = (corner: CornerType, isSolid: boolean) => dcnb(
-  'su-w-[83%] su-h-full su-border-t-2 su-border-l-2 su-rounded-tl-full',
+export const curve = (corner: CornerType, isSolid: boolean) => cnb(
+  'su-w-[83%] su-h-full',
```
Comment (yvonnetangsu) [inline src/components/Bracket/Bracket.styles.ts:15]: Remove the dupe classes (it was unintentional) here since we aren't using dedupe anymore

**Comment by sherakama** [comment]
Lol. Gave us 2 minutes to review before pulling the trigger?

**Comment by yvonnetangsu** [comment]
> Lol. Gave us 2 minutes to review before pulling the trigger?

Hahaha I didn't think you all would actually have time to review 😂 ! I'll leave them up for longer next time 👍🏼 🤣

**Comment by sherakama** [comment]
Did you measure the difference between `cnb` and `dcnb`? It is supposed to be faster but I am wondering if you ran in to something to trigger this optimization here?

**Comment by yvonnetangsu** [comment]
> Did you measure the difference between `cnb` and `dcnb`? It is supposed to be faster but I am wondering if you ran in to something to trigger this optimization here?

Good question, @sherakama ! 😄  So nope, nothing really trigger this. Just that I don't find us using dedupe at all due to the way we use cnbuilder, so thought why don't I just go with the faster one.

I didn't do benchmarking myself, but comparing the author's results (screenshot below). the cnb version definitely seems more performant. I doubt it's noticeable for us but hey why not 🤷🏼‍♀️ ? With how much animation we're putting in, every little bit helps I guess?

cnb:

Screenshot 2023-06-09 at 12 06 41 PM


dcnb:

Screenshot 2023-06-09 at 12 07 19 PM

**Comment by sherakama** [comment]
Sounds logical to me. Thanks for sharin'

**Comment by yvonnetangsu** [comment]
I ran into a Reddit comment and someone mentioned `classname` vs `clsx` (which is similar to `cnbuilder` but still slower), which made me want to take a look at this again 😄 @sherakama

**Pull request merged**